src/components/Tabset/__test__/tabset.spec.js

```diff
@@ -14,6 +31,22 @@ describe('<Tabset />', () => {
         );
         expect(component.find('p').text()).toBe('testing tabset');
     });
+    it('should have the right class names', () => {
+        const component = mount(
+            <Tabset className="my-custom-class-name">
+                <Tab label="Tab-1" name="tab-1" registerTab={registerTabMockFn} />
+            </Tabset>,
+        );
+        expect(component.find('div').prop('className')).toBe('rainbow-tabset my-custom-class-name');
```

Comment:
this is to fragile since if in the future we add some div inside the tabset or tab then likely it will break, in this case better use:
```expect(component.find('.rainbow-tabset my-custom-class-name').exists()).toBe(true);```